committee-steering/governance/sig-governance.md

```diff
@@ -160,8 +160,7 @@ within 30 days from the date of their appointment.
   - *MUST* be a contact point for the Product Security Committee to reach out to
    for triaging and handling of incoming issues
   - *MUST* accept the [Embargo Policy]
-  - Defined in `SECURITY_CONTACTS` files, this is only relevant to the root file
-   in the repository. Template [SECURITY_CONTACTS]
+  - Defined in `security_contacts` field of `OWNERS` files
```

Comment:
Should we add a note that if no `security_contacts` are specified, we default to the top-level project approvers?

Reply:
Added